server/config/routes/campaigns.js

```diff
@@ -7,6 +7,6 @@ const router = express.Router();
 router.use(passport.authenticate('jwt', { session: false }));
 
 router.route('/').post(saveNewCampaign);
-router.route('/').get(getAllCampaigns);
+router.route('/:status').get(getAllCampaigns);
```

Comment:
So we don't want to use a path param, but instead use a query param.
A path param looks like /path/to/some_resource/:status where status is now a path param.
A query param looks like /path/to/some_resource?status='active' where status is now a query param. Does that make sense?

http://expressjs.com/en/api.html#req.query shows how to accomplish this

Reply:
Ohh okay. Yes, that makes sense. I'll work on fixing this.